index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>IsaMarvin | Software Engineer</title>
+  </head>
+  <body>
+    <header class="header">
+      <img src="./images/my-logo.svg" alt="logo" />
+      <img
+        src="./images/hamberger.svg"
+        alt="hamburger"
+      />
+    </header>
+
+    <section class="wrapper">
+      <div class="hero">
+        <h1 class="hero__title">I’m IsaMarvin. Glad to see you!</h1>
+        <p class="hero__description">
+          I’m a software developer! I can help you build a product, feature, or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <p class="hero__connect">LET’S CONNECT</p>
+        <div class="social">
+          <a href="#" class="social__link">
+            <img
+              class="social__icon"
+              src="./images/social/twitter.svg"
+              alt="twitter"
+            />
+          </a>
+          <a href="#" class="social__link">
+            <img
+              class="social__icon"
+              src="./images/social/linkedin.svg"
+              alt="linkedin"
+            />
+          </a>
+          <a href="#" class="social__link">
+            <img
+              class="social__icon"
+              src="./images/social/medium.svg"
+              alt="medium"
+            />
+          </a>
+          <a href="#" class="social__link">
+            <img
+              class="social__icon"
+              src="./images/social/github.svg"
+              alt="github"
+            />
+          </a>
+          <a href="#" class="social__link">
+            <img
+              class="social__icon"
+              src="./images/social/vector.svg"
+              alt="angellist"
+            />
+          </a>
+        </div>
+      </div>
```

Comment:
- You have done well so far, but to improve the semantics of your page, I suggest you wrap this section in a `ul` tag and wrap all `a` tags in a corresponding `li` tag. Please fix this 👍 👍